applications/oci-transaction-lifecycle.md

```diff
@@ -7,21 +7,21 @@
 
 ### Project Details
 
-- **A. Transaction Retry:** API nodes will monitor their transactions as they are sent into the system and ensure they are resubmitted into the system if they are not processed in a configurable timeperiod. This feature will establish a pool of known incoming transactions and monitor their inclusion into the blockchain. Once the system reaches an acceptable level of confidence that a transaction has been included in a block, the transaction can be pruned from the pool. If a transaction is identified as missing from the blockchain based on the given criteria, it will attempt to resubmit the transaction to the network for inclusion in future blocks until the point it expires.
-- **B. Transaction Finality Status:** API nodes will monitor their transactions as they are sent into the system and will provide a new API method to report transaction status.
-- **C. Transaction Resource Cost Estimation:** Nodes will provide an estimate of resources (central processing unit (CPU), random access memory (RAM), and internet bandwidth (NET)) to perform a transaction when a transaction is sent to new compute_transaction endpoint.  The transaction will be applied subjectively to the local chain to calculate resource costs like CPU and NET and also determine the resulting deltas in RAM usage. The response will be returned to the client to inform them of the costs associated with the given transaction data.
-- **D. Subjective Billing Improvements:** Nodes will provide an added leeway per account to allow a more permissive “subjective billing” rate (rate of utilization) to provide a better user experience while maintaining system integrity. The existing subjective CPU decay window will be made configurable from the existing hard-coded value of 24 hours to allow node opperators to provide a more relaxed subjective CPU penalty to their users. The existing `disable-subjective-account-billing` will be expanded to also apply to the 3-strike rule allowing block producers to prevent abnormal transaction loss for accounts expected to sometimes fail during production.
+- **A. Transaction Retry:** API nodes will monitor transactions as they enter the system and ensure they are resubmitted into the system if they are not processed in a configurable time period. This feature will establish a pool of known incoming transactions and monitor their inclusion into the blockchain. Once the system reaches an acceptable level of confidence that a transaction has been included in a block, the transaction can be pruned from the pool. If a transaction is identified as missing from the blockchain based on the given criteria, the Transaction Retry feature will attempt to resubmit the transaction to the network for inclusion in future blocks until the point at which the transaction expires.
```

Comment:
I wasn't sure what noun "it" referred to in "until the point it expires." I assumed it was the transaction. But it might also have meant the time period. So please make sure I edited this correctly.

Reply:
you are correct,"it" was transaction